cli/src/template/ios/MoproApp/ContentView.swift

```diff
@@ -85,9 +85,10 @@ extension ContentView {
             
             // Generate Proof
             let generateProofResult = try generateCircomProof(zkeyPath: zkeyPath, circuitInputs: input_str, proofLib: ProofLib.arkworks)
-            assert(!generateProofResult.proof.isEmpty, "Proof should not be empty")
-            assert(Data(expectedOutput) == generateProofResult.inputs, "Circuit outputs mismatch the expected outputs")
-            
+            let public_inputs = serializeOutputs(generateProofResult.inputs)
```

Comment:
Ideally we can just compare the output string `generateProofResult.inputs` and the expected string e.g. 
```swift
let outputs: [String] = [String(c), String(a)]
```
so it won't convert to `serializedProof` either
and it will be more readable